mpcstats_lib.py

```diff
@@ -38,12 +38,33 @@ def print_data(data: Matrix):
             print_ln("data[{}][{}]: %s".format(i, j), data[i][j].reveal())
 
 
+def _mean(data: list[sint]) -> (float, int):
+    total = sum(if_else(i != MAGIC_NUMBER, i, 0) for i in data)
+    count = sum(if_else(i != MAGIC_NUMBER, 1, 0) for i in data)
+    return total / count, count
+
+
+def _variance(data: list[sint], use_bessels: bool) -> float:
+    # calculate mean of the data excluding magic numbers
+    mean, eff_size = _mean(data)
+
+    # replace magic number w/ mean to skip magic number in for loop below
+    data = Array.create_from(if_else(n != MAGIC_NUMBER, n, mean) for n in data)
+
+    eff_data_sum = sfloat(0)
```

Comment:
I think we should replace all `sfloat` with `sfix`
```suggestion
    eff_data_sum = sfix(0)
```

Reply:
What I have observed while working on this was that if I use sfix the accuracy goes down slightly. But now I think this can be resolved after we improve the precision of sfix. I made the change.